sphinx/texinputs/Makefile_t

```diff
@@ -57,30 +87,8 @@ bz2: tar
 xz: tar
 	xz -9 -k $(ARCHIVEPREFIX)docs-$(FMT).tar
 
-# The number of LaTeX runs is quite conservative, but I don't expect it
-# to get run often, so the little extra time won't hurt.
-%.dvi: %.tex
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-	-$(MAKEINDEX) -s python.ist '$(basename $<).idx'
-	$(LATEX) $(LATEXOPTS) '$<'
-	$(LATEX) $(LATEXOPTS) '$<'
-
-%.pdf: %.tex
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	-$(MAKEINDEX) -s python.ist '$(basename $<).idx'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-	$(PDFLATEX) $(LATEXOPTS) '$<'
-
-%.ps: %.dvi
-	dvips '$<'
-
 clean:
-	rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz $(ALLPDF) $(ALLDVI)
+	rm -f *.log *.ind *.aux *.toc *.syn *.idx *.out *.ilg *.pla *.ps *.tar *.tar.gz *.tar.bz2 *.tar.xz $(ALLPDF) $(ALLDVI) $(ALLXDV) *.fls *.fdb_latexmk
```

Comment:
I do not use ``$(ALLDVI)$(ALLXDV)`` which avoids superfluous space because it would make mandatory that always one of them is empty, possibly creating error in future maintenance of this. For the record `.xdv` files are created by `xelatex` runs; if `xelatex` is run manually, the `xdv` file is removed but here it is run by Latexmk rule deliberately to not produced `pdf` until last final `xdvipdfmx`, for efficiency. So the `xdv` file remains in build repertory. (like `dvi` file when doing `platex+dvipdfmx`).

Side remark on legacy situation: this ``rm`` clearly may have unintended effects if ``$(ALLPDF)`` expands to filenames with spaces. Brief investigation on the net teaches me GNU Make at least is very awkward with spaces in filenames. Hence, Sphinx does not support here spaces in (basenames of) filenames of base documents. The shell globbing does not have this issue, but I think names coming from the Make variable have it. Anyway, this ``clean`` is for Users who actually ``cd`` to build repertory, which is not needed thanks to ``make latexpdf`` at top level, and ``make clean`` there is something else.